sysvabi64/sysvabi64.rst

```diff
@@ -1592,6 +1592,163 @@ IFUNC requirements for dynamic linkers
+    +-------------------+----------+
+    | FEAT_PMULL        | 1U << 16 |
+    +-------------------+----------+
+    | FEAT_FP16         | 1U << 17 |
+    +-------------------+----------+
+    | FEAT_DIT          | 1U << 18 |
+    +-------------------+----------+
+    | FEAT_DPB          | 1U << 19 |
+    +-------------------+----------+
+    | FEAT_DPB2         | 1U << 20 |
+    +-------------------+----------+
+    | FEAT_JSCVT        | 1U << 21 |
+    +-------------------+----------+
+    | FEAT_FCMA         | 1U << 22 |
+    +-------------------+----------+
+    | FEAT_RCPC         | 1U << 23 |
+    +-------------------+----------+
+    | FEAT_RCPC2        | 1U << 24 |
+    +-------------------+----------+
+    | FEAT_FRINTTS      | 1U << 25 |
+    +-------------------+----------+
+    | FEAT_I8MM         | 1U << 27 |
+    +-------------------+----------+
+    | FEAT_BF16         | 1U << 28 |
+    +-------------------+----------+
+    | FEAT_SVE          | 1U << 31 |
+    +-------------------+----------+
+    | FEAT_SVE_F32MM    | 1U << 35 |
+    +-------------------+----------+
+    | FEAT_SVE_F64MM    | 1U << 36 |
+    +-------------------+----------+
+    | FEAT_SVE2         | 1U << 37 |
+    +-------------------+----------+
+    | FEAT_SVE_PMULL128 | 1U << 39 |
+    +-------------------+----------+
+    | FEAT_SVE_BITPERM  | 1U << 40 |
+    +-------------------+----------+
+    | FEAT_SVE_SHA3     | 1U << 41 |
+    +-------------------+----------+
+    | FEAT_SVE_SM4      | 1U << 42 |
+    +-------------------+----------+
+    | FEAT_SME          | 1U << 43 |
+    +-------------------+----------+
+    | FEAT_MEMTAG2      | 1U << 45 |
+    +-------------------+----------+
+    | FEAT_SB           | 1U << 47 |
+    +-------------------+----------+
+    | FEAT_SSBS2        | 1U << 50 |
+    +-------------------+----------+
+    | FEAT_BTI          | 1U << 51 |
+    +-------------------+----------+
+    | FEAT_WFXT         | 1U << 55 |
+    +-------------------+----------+
+    | FEAT_SME_F64      | 1U << 56 |
+    +-------------------+----------+
+    | FEAT_SME_I64      | 1U << 57 |
+    +-------------------+----------+
+    | FEAT_SME2         | 1U << 58 |
+    +-------------------+----------+
+    | FEAT_RCPC3        | 1U << 59 |
+    +-------------------+----------+
+    | FEAT_MOPS         | 1U << 60 |
+    +-------------------+----------+
+
+A special value ``FEAT_INIT = (1U << 63);`` is used to signify
+initialization completion.
+
+Implementing FMV using ``__aarch64_cpu_features`` is not required.
+Accessing ``__aarch64_cpu_features`` from outside a FMV resolver
+function is not well defined.
+
+Initialization of the ``__aarch64_cpu_features`` is performed by
+the runtime library. Each FMV resolver function will call the
+``__init_cpu_features_resolver`` function at load time. This runtime
+function must ensure that ``__aarch64_cpu_features`` is populated.
+
+Systems without support for IFUNC may call the
```

Comment:
If compiler's are always going to use IFUNCs for a SystemV target such as `--target=aarch64-linux-gnu` or `--target=aarch64-linux-android` then a SystemV platform that doesn't support IFUNCs the dynamic loader is just going to reject the binary at dynamic load time as it won't know what to do with the IFUNC relocations.

I think we should take this part out unless we have support in SystemV platforms for an alternative strategy that doesn't use IFUNCs.